.travis.yml

```diff
@@ -3,9 +3,7 @@ language: java
 matrix:
   include:
   - dist: precise
-    jdk: openjdk7
-  - dist: precise
-    jdk: oraclejdk7
+    jdk: oraclejdk8
```

Comment:
This is unnecessary since we already have oraclejdk8 in the test matrix (See line 8)